docs/guides/volume-populator/pvc/index.md

```diff
@@ -0,0 +1,451 @@
+
+Verify created `Snapshot` object by the following command,
+
+```bash
+$ kubectl get snapshots -n demo
+NAME                                                       REPOSITORY       SESSION           SNAPSHOT-TIME          DELETION-POLICY   PHASE       VERIFICATION-STATUS   AGE
+gcs-repository-pvc-backup-frequent-backup-1704281100       gcs-repository   frequent-backup   2024-01-03T11:25:13Z   Delete            Succeeded                         2m14s
+```
+
+> When a backup is triggered according to schedule, KubeStash will create a `Snapshot` with the following labels  `kubestash.com/app-ref-kind: PersistentVolumeClaim`, `kubestash.com/app-ref-name: <volume-name>`, `kubestash.com/app-ref-namespace: <volume-namespace>` and `kubestash.com/repo-name: <repository-name>`. We can use these labels to watch only the `Snapshot` of our desired Workload or `Repository`.
+
+## Populate Volume
+
+This section will show you through the process of populating a volume by restoring the previously backed-up data using KubeStash.
+
+**Create a Stand-alone PVC:**
+
+Now, we are going to create a stand-alone `PVC` with `spec.dataSourceRef` set and refers our `snapshot` object.
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: populate-pv
+  namespace: demo
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+  dataSourceRef:
+    apiGroup: storage.kubestash.com
+    kind: Snapshot
+    name: gcs-repository-nfs-pvc-backup-frequent-backup-1704281100
+```
+Here,
+  - `spec.dataSourceRef` specifies that which `snapshot` we want to restore into our populate volume.
+  - we are referencing a `snapshot` object that we have backed up in the previous section.
+
+Let's create the `PVC` object that we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/volume-populator/pvc/examples/populate-pv.yaml
+persistentvolumeclaim/populate-pv created
+```
+
+**Wait for Populate Volume:**
+
+When you create a `PVC` with `spec.dataSourceRef` set and refers our `snapshot` object, KubeStash will create a restore Job. Now, wait for the volume populate process to complete. 
+
+You can watch the `PVC` status using the following command,
+
+```bash
+$ watch -n 1 kubectl get pvc -n demo populate-pv
+
+Every 1.0s: kubectl get pvc -n demo populate-pv                                workstation: Wed Jan  3 17:30:20 2024
+
+NAME                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+populate-pv         Bound    pvc-8d509c47-c26e-4039-8c03-49b25d126429   2Gi        RWO            standard       5d1
+```
+From the output of the above command, we can see that `PVC` status is `Bound` that means populate volume has been completed successfully.
```

Comment:
The output of the command shows the PVC status as Bound, indicating successful completion of the volume population.